src/components/molecules/NavbarSchedule/NavbarSchedule.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { useCallback, useState } from "react";
+import classNames from "classnames";
+
+import { SchedulePageModal } from "components/organisms/SchedulePageModal/SchedulePageModal";
+import { useSelector } from "hooks/useSelector";
+import { currentVenueSelectorData, parentVenueSelector } from "utils/selectors";
+
+export const NavbarSchedule = () => {
+  const venue = useSelector(currentVenueSelectorData);
+  const parentVenue = useSelector(parentVenueSelector);
+
+  const [isEventScheduleVisible, setEventScheduleVisible] = useState(false);
+
+  const toggleEventSchedule = useCallback(() => {
+    setEventScheduleVisible(!isEventScheduleVisible);
+  }, [isEventScheduleVisible]);
+
+  const hideEventSchedule = useCallback(() => {
+    setEventScheduleVisible(false);
+  }, []);
+
+  const navbarTitle = parentVenue?.name ?? venue?.name;
```

Comment:
This seems to have lost the comment that was previously attached to it, why is that?

```
// TODO: ideally this would find the top most parent of parents and use those details
```

Reply:
Added back.